### ISSUE TITLE
There is no method to skip repetitive content at the top of the website pages

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1936]
URL: http://clarity-ks.surge.sh/alerts

Reproduced Steps:
1. Go to Alerts page.
2. Try to skip repetitive content at the top of the website pages.

Actual Results:
There is no method to skip repetitive content at the top of the website pages.
Please refer to “Repetitive content.png” to see the repetitive content.

Expected Results:
There should be a method to skip repetitive content at the top of the website pages.

Violation of Section 508 Standards: 
1194.22 (o) A method shall be provided that permits users to skip repetitive navigation links. 
Screen shot:

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/4ab66cb1-840f-44ed-8d44-70b7f5088c47)

**Comment by Jinnie** [comment]
I don't understand the requirements fully. 
The way our navigation is implemented right now, it gets focus only after the user tabs through the content. If we want to perform classical "skip to content" hidden item, then we first need to make our nav links get focus before the content.

**Comment by mathisscott** [comment]
@Jinnie 
The vendor expected us to have a skip link. [More information here](https://webaim.org/techniques/skipnav/).

**Comment by Jinnie** [comment]
I understand this, but as they say on line 1 of their document "The main content is not usually the first thing on a web page."
The _problem_ is that the main content _IS_ before the nav on our site. I can probably change it, just to satisfy the scenario, but only if it gets projected directly, without a selector. Otherwise, it will be like:
1. Hello.
2. Goto 3.
3. Show content.
4. Nav is here.

**Comment by mathisscott** [comment]
They are just looking for a skip link. This is a VPAT vendor. They literally run a program and then copy-paste what the program tells them. If you feel a skip link would be overkill (and it's clear that you do), then respond as such in this thread and then close this issue with a "wontfix" label.

That is an acceptable outcome.

We are not reordering our layout for this. It's not worth it. Thanks.

**Comment by Jinnie** [comment]
I'll make it with a minor change on the app itself, to make it meaningful, like this:
1. Hello.
2. Goto 4 (skip-link)
3. Nav (up from 4.)
4. Show content (down from 3.)

We may do the same on the website, possibly? Now it also follows the scenario from my previous comment.

**Comment by mathisscott** [comment]
I would keep this change to just the kitchen sink app. Feel free to open another issue for the website. The goal of these tickets is to prepare the kitchen sink app for another round of VPAT testing.
